colophon.rst

```diff
@@ -0,0 +1,56 @@
+Colophon
+===================
```

Comment:
```suggestion
Colophon
========
```

Reply:
I wouldn't normally bother tidying up longer underlines, but since you've gone to the trouble, thanks.